html_sanitizer/sanitizer.py

```diff
@@ -112,17 +112,19 @@ def anchor_id_to_name(element):
     return element
 
 
-def normalize_whitespace_in_text_or_tail(element, whitespace_re=None):
+def normalize_whitespace_in_text_or_tail(
+    element, whitespace_re=None, keep_whitespaces=None
```

Comment:
I think I'd reuse the `keep_typographic_whitespace` name here (and everywhere else).

Also, `None` doesn't seem like a good default; the default should probably be `False`.

Reply:
Done!